src/packages/StorageFileApi.ts

```diff
@@ -258,10 +258,12 @@ export default class StorageFileApi {
    *
    * @param path The file path, including the current file name. For example `folder/image.png`.
    * @param expiresIn The number of seconds until the signed URL expires. For example, `60` for a URL which is valid for one minute.
+   * @param options customizes the public url with query string params
```

Comment:
I think a better to document this for it to show up in our docs is to do 
```
@param options.download triggers the file as a download the file if set to true. Set this parameter as the name of the file if you want to trigger the download with a different filename.
```

Reply:
@inian Fixed